security/oidc-client-mutual-tls/src/test/resources/openshift-rh-sso-deployment-template.yml

```diff
@@ -0,0 +1,77 @@
+---
+apiVersion: "v1"
```

Comment:
I think that would be great to add this template to [Quarkus-test-framework](https://github.com/quarkus-qe/quarkus-test-framework) so based on `KeycloakService` configuration the framework is able to deploy keycloak with Mtls or a regular one. Please add this feature as part of the previously created issue

Reply:
In order to do that, I'll need to adjust at the very least `io.quarkus.test.services.containers.OpenShiftContainerManagedResource#replaceDeploymentContent` as `tls.crt` and `tls.key` should be configurable if we add ths template to Quarkus-test-framework. I will do this change together with the other issue.